Utils/API/server/setup.sh

```diff
@@ -19,6 +19,11 @@ NGINX_GROUP=
 MANAGE_SERVICE=
 MANAGE_SEL=
 
+# Change with caution, for this directory must have
+# specific permissions that allow nginx user
+# access to the socket files
+SOCK_DIR="$RUN_DIR"
```

Comment:
RUN_DIR is default here. If RUN_DIR is changed via command line argument later, SOCK_DIR remains the same. Is this OK?

Reply:
Yes, that's the exact point: `SOCK_DIR` must be readable and writable for Nginx user, so changing it makes sence only if you're sure that you have privileges to allow it to the desired directory. Of course, no one will stop you from making some directory read/writable for everyone, but it won't be very good idea -- since it means that anyone can (even erroneously) remove your app socket file, for example. Or write to the socket something, or read, whatever.

But you have a point: if we want specific value for this variable, it is a bit wierd to define it with value of another one. I think I better set `SOCK_DIR` to the desired value and `RUN_DIR` to `$SOCK_DIR` -- just to make sure that it won't be changed accidentally.